### PR TITLE
chore(upgrade): update upgrade README with 1.9.0 steps

### PR DESCRIPTION
**Pull request by shubham14bajpai**
Signed-off-by: shubham <shubham.bajpai@mayadata.io>

This PR updates the README with bulk upgrade option and 1.9.0 steps.

### CODE REVIEW

File: k8s/upgrades/README.md [hunk 1]
```diff
@@ -175,17 +177,20 @@ using Kubernetes Job spec.
 
 The following instructions provide details on how to create your Upgrade Job specs.
 Please ensure the `from` and `to` versions are as per your upgrade path. The below
-examples show upgrading from 1.0.0 to 1.8.0.
+examples show upgrading from 1.0.0 to 1.9.0.
 
 ### Upgrade the OpenEBS Jiva PV
 
-**Note: Scaling down the application will speed up the upgrade process. It is highly recommended to scale down the application if upgrading from 1.5.0 or earlier versions of the volume.**
+**Note:Scaling down the application will speed up the upgrade process and prevent any read only issues. It is highly recommended to scale down the application if upgrading from 1.8.0 or earlier versions of the volume.**
```
Comment (vishnuitta) [inline k8s/upgrades/README.md:184]: what is the point of mentioning 'if upgrading from 1.8.0 or earlier versions of the volume'? we can remove that 'if'

File: k8s/upgrades/README.md [hunk 2]
```diff
@@ -226,10 +231,14 @@ spec:
         - "--from-version=1.0.0"
 
         # --to-version is the version desired upgrade version
-        - "--to-version=1.8.0"
+        - "--to-version=1.9.0"
 
-        #VERIFY that you have provided the correct cStor PV Name
+        # VERIFY that you have provided the correct jiva PV Name
```
Comment (vishnuitta) [inline k8s/upgrades/README.md:236]: you can say that this gets deprecated

File: k8s/upgrades/README.md [hunk 3]
```diff
@@ -226,10 +231,14 @@ spec:
...
         - "--pv-name=pvc-713e3bb6-afd2-11e9-8e79-42010a800065"
+        # From 1.9.0 onwards the pv-name flag is not required and
```
Comment (vishnuitta) [inline k8s/upgrades/README.md:238]: you can mention just this one, and outside of this sample yaml, mention that '--pv-name' is also supported for backward compatibility for few more releases

Comment (vishnuitta) [reply k8s/upgrades/README.md:238]: same comment applies for other places also
Also, need to mention how to read the job output to know the status of individual volume's upgrade

File: k8s/upgrades/README.md [hunk 4]
```diff
@@ -226,10 +231,14 @@ spec:
         - "--from-version=1.0.0"
 
         # --to-version is the version desired upgrade version
-        - "--to-version=1.8.0"
+        - "--to-version=1.9.0"
 
-        #VERIFY that you have provided the correct cStor PV Name
+        # VERIFY that you have provided the correct jiva PV Name
         - "--pv-name=pvc-713e3bb6-afd2-11e9-8e79-42010a800065"
```
Comment (kmova) [inline k8s/upgrades/README.md:237]: comment this line, in favor of the below format to specify multiple volumes.

File: k8s/upgrades/README.md [hunk 5]
```diff
@@ -381,10 +395,13 @@ spec:
         - "--from-version=1.0.0"
 
         # --to-version is the version desired upgrade version
-        - "--to-version=1.8.0"
+        - "--to-version=1.9.0"
 
         #VERIFY that you have provided the correct cStor PV Name
         - "--pv-name=pvc-c630f6d5-afd2-11e9-8e79-42010a800065"
+        # From 1.9.0 onwards the pv-name flag is not required and
```
Comment (kmova) [inline k8s/upgrades/README.md:402]: Same comment like above applies here. Comment the line with --pv-name.

File: k8s/upgrades/README.md [hunk 6]
```diff
@@ -308,10 +319,15 @@ spec:
         - "--from-version=1.0.0"
 
         # --to-version is the version desired upgrade version
-        - "--to-version=1.8.0"
+        - "--to-version=1.9.0"
 
-        #VERIFY that you have provided the correct SPC Name
-        - "--spc-name=cstor-sparse-pool"
+        # VERIFY that you have provided the correct SPC Name
+        # The spc-name flag is required till 1.8.0 upgrades
```
Comment (vishnuitta) [inline k8s/upgrades/README.md:325]: Lets add comment as:
# Bulk upgrade is supported from 1.9
# To make use of it, please provide the list of SPCs
# as mentioned below
- "cstor-sparse-pool"
- "cstor-disk-pool"
# For upgrades lesser than 1.9.0, use
# '--spc-name=<spc_name> format as
# below commented line
# - "--spc-name=cstor-sparse-pool"

**Pull request merged**